mozregression/fetch_configs.py

```diff
@@ -269,6 +270,24 @@ def tk_inbound_route(self, changeset):
         )
 
 
+class B2GDeviceConfigMixin(InboundConfigMixin):
+    inbound_branch = 'b2g-inbound'
+
+    def set_build_type(self, build_type):
+        self.build_type = build_type
+
+    def tk_inbound_route(self, changeset):
+        print "Querying inbound route: 'gecko.v2.{}.revision.{}.{}'".format(
+            self.inbound_branch, changeset, self.build_type,
+        )
+        return 'gecko.v2.{}.revision.{}.{}'.format(
+            self.inbound_branch, changeset, self.build_type,
+        )
+
+    def set_nightly_repo(self, repo):
```

Comment:
hm, I suspect you had to do that because we never had a case with an inbound only mixin. We should fix that in a followup, but for now can you move that method into B2GDeviceConfig (since this is for nightly, it would make more sense)